Editor/MenuActions/Editors/NewBezierShape.cs

```diff
@@ -26,7 +26,7 @@ public override bool hidden
 
         public override bool enabled
         {
-            get { return true; }
+            get { return Experimental.experimentalFeaturesEnabled && ProBuilderEditor.instance != null; }
```

Comment:
@karl-  I would also love to add something like :
        `protected override bool hasFileMenuEntry { get { return Experimental.experimentalFeaturesEnabled; } }`
For the Bezier shape, but that would involved to rebuild the Menu entries when switching experimental mode on/off which is there not possible (Experimental not aware of the Debug classes in which GenerateMenuItems class is)